packages/react-core/src/demos/ComposableMenu/examples/DateSelectDemo.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { MenuToggle, Select, SelectList, SelectOption } from '@patternfly/react-core';
+
+export const DateSelectDemo: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selected, setSelected] = React.useState<number>(0);
+  const menuRef = React.useRef<HTMLDivElement>();
+
+  const onToggleClick = (ev: React.MouseEvent) => {
+    ev.stopPropagation(); // Stop handleClickOutside from handling
+    setTimeout(() => {
+      if (menuRef.current) {
+        const firstElement = menuRef.current.querySelector('li > button:not(:disabled)');
+        firstElement && (firstElement as HTMLElement).focus();
+      }
+    }, 0);
+    setIsOpen(!isOpen);
+  };
```

Comment:
Similar to above comment

```suggestion
  const onToggleClick = (ev: React.MouseEvent) => {
    setIsOpen(!isOpen);
  };
```